enable/vtk_backend/vtk_window.py

```diff
@@ -281,6 +283,7 @@ def _vtk_mouse_wheel(self, vtk_obj, eventname):
             self._pass_event_to_vtk(vtk_obj, eventname)
 
     def _create_key_event(self, vtk_event, event_type):
+        # FIXME: THIS IS A BUG
```

Comment:
No, I don't.

`_create_key_event` is called from `AbstractWindow`, but the way the events are being routed from VTK here will throw an exception before `_create_key_event` can be called. The `FIXME` comment is fine for now.